test/laws.jl

```diff
@@ -0,0 +1,166 @@
+# Defining dumb inputs is easier than creating a whole `Simulation` object for testing `generate_input`
+module DumbTestInputs
```

Comment:
I would suggest changing "dumb" by "dummy" or any other name that implies that this is a simple test. The word "dumb" may sound a bit offensive in English!